File: FLM_SubconLabelSystem/Library/Library.Database/BLL/HouseKeep.cs
using System.Data;

namespace Library.Database.BLL
{
    /// <summary>
    /// Business Logic Layer
    /// ---------------------------------
    /// 18 Feb 2012   Yeon    Initial Version
    /// </summary>
    public class HouseKeep : Library.Root.Other.BusinessLogicBase
    {
        public static DataTable GetSubSlitChild(string Company, string datePurge, string pPurgeTable)
        {
            using (var _dal = new DAL.HouseKeep())
            {
                return _dal.GetSubSlitChild(Company, datePurge, pPurgeTable);
            }
        }

        public static string DelSubSlitChild(string pID, string pHKTable)
        {
            using (var _Dal = new DAL.HouseKeep())
            {
                string result = _Dal.DelSubSlitChild(pID, pHKTable);

                if (result == "1")
                {
                    _Dal.Commit();
                }
                else
                {
                    _Dal.Rollback();
                }
                return result;
            }
        }
    }
}

File: FLM_SubconLabelSystem/Library/Library.Database/BLL/HouseKeep.vb
Namespace BLL
    ''' <summary>
    ''' Business Logic Layer
    ''' ---------------------------------
    ''' 18 Feb 2012   Yeon    Initial Version
    ''' </summary>
    ''' <remarks></remarks>
    Public Class HouseKeep
        Inherits Library.Root.Other.BusinessLogicBase

        Public Shared Function GetSubSlitChild(ByVal Company As String, ByVal datePurge As String, ByVal pPurgeTable As String) As DataTable
            Using _dal As New DAL.HouseKeep
                GetSubSlitChild = _dal.GetSubSlitChild(Company, datePurge, pPurgeTable)
            End Using
        End Function

        Public Shared Function DelSubSlitChild(ByVal pID As String, ByVal pHKTable As String) As String
            Using _Dal As New DAL.HouseKeep

                DelSubSlitChild = _Dal.DelSubSlitChild(pID, pHKTable)

                If DelSubSlitChild = "1" Then
                    _Dal.Commit()
                Else
                    _Dal.Rollback()
                End If
            End Using
        End Function
    End Class
End Namespace

